docs/src/main/sphinx/release/release-419.md

```diff
@@ -0,0 +1,41 @@
+# Release 419 (24 May 2023)
+
+## General
+
+* Add the {func}`array_histogram` function to find the number of occurrences of
+  the unique elements in an array. ({issue}`14725 `)
+* Fix JSON serialization failure when using string columns with values larger
+  than 5 MB. ({issue}`17557`)
+
+## Delta Lake connector
+
+* Improve performance of reading from Parquet files. ({issue}`17612`)
+* Improve display of table handles in the output of `EXPLAIN` queries. ({issue}`17590`)
```

Comment:
"Table handle" is an internal concept. How is the display improved? Describe in terms of user-visible impact. cc @losipiuk